autoload/gin/internal/component.vim

```diff
@@ -12,13 +12,17 @@ function! gin#internal#component#init(component) abort
     execute printf('autocmd User GinCommandPost call gin#internal#component#update("%s")', a:component)
     execute printf('autocmd User DenopsPluginPost:gin call gin#internal#component#update("%s")', a:component)
   augroup END
+  call gin#internal#component#update(a:component)
```

Comment:
Why do we need this?

Reply:
It fix didn't call update until next BufEnter when called after plugin initialized.

Reply:
Will can call this timing at this fix, so I did, but I thought shouldn't mixed it. delete from this PR.